pkg/operator/ceph/cluster/controller.go

```diff
@@ -719,6 +719,10 @@ func (c *ClusterController) onDeviceCMUpdate(oldObj, newObj interface{}) {
 			logger.Infof("Cluster %s is not ready. Skipping orchestration on device change", cluster.Namespace)
 			continue
 		}
+		if len(cluster.Spec.Storage.StorageClassDeviceSets) > 0 {
+			logger.Debugf("skip orchestration on device config map update for OSDs on PVC")
```

Comment:
```suggestion
			logger.Infof("skip orchestration on device config map update for OSDs on PVC")
```

This seems like a useful message given that we `continue`.

Reply:
Also, to be honest, I don't understand why checking the length of `cluster.Spec.Storage.StorageClassDeviceSets` solves your problem.

Reply:
having `StorageClassDeviceSets` could ensure that cluster (in the current loop iteration) is of type `OSD on PVC` (and not `OSD on device`) .  So we can skip starting the orchestration again.

Reply:
I see, makes sense now, thanks.